### DISCUSSION TITLE
Is there a way to obtain the date/timestamp of the last pull performed on a repository ?

### DISCUSSION THREAD

**Discussion by PyAxolotl** [community/community#64080] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hi, 

I'm searching to find a way to determine if a repository is still being cloned/pulled. I found this API request that allows to get clones recent history https://docs.github.com/en/rest/metrics/traffic?apiVersion=2022-11-28#get-repository-clones but I didn't find a way to retrieve the history of pulls done. 

Does some know if it's possible to retrieve this information ?

**Comment by byrneh** [comment]
Hi @PyAxolotl is REST API [#list-pull-requests](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests) what you are looking for?

**Comment by byrneh** [comment, marked as answer]
Not seen somethig that will show a "git pull"
If you are uaing a GitHub plan that has access to GitHub Audit Log API, that also provides some audit entries related to git activity [#git-category-actions](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/audit-log-events-for-your-organization#git-category-actions)

**Comment by antonio-pedro99** [comment]
If you want to know if people are sending pull requests, you can try search with 
`is:pr created:>2023-08-10` this will list all pr created after 2023-08-10

**Reply by antonio-pedro99** [reply]
Optionally, you want to check if it has interactions or reviews 
`interactions:>5`

**Comment by PyAxolotl** [comment]
@byrneh @antonio-pedro99 Hi guys, thanks for the answers. As you mentioned @byrneh I'm trying to find a way to access history of **git pulls** and **git fetch** done on a repo and I do not have a "GitHub Enterprise Cloud" to access the Audit log so I was wondering if there was another way.

**Reply by byrneh** [reply]
@PyAxolotl in that case unfortunately as far as I know you are out of luck with obtaining the pull details you are looking for

**Reply by PyAxolotl** [reply]
@byrneh Yes, I raised a GitHub support ticket and that's what they pretty much said to me. They pointed me to the Insights tab on repositories which only shows views and clones. Overall I think the GitHub Audit Log API is the way to go so the upgrade of the account is mandatory to be able to access that data.
Thanks again,
Have a nice one
